test/functional/feature_csv_activation.py

```diff
@@ -103,12 +102,14 @@ def create_self_transfer_from_utxo(self, input_tx):
     def create_bip112special(self, input, txversion):
         tx = self.create_self_transfer_from_utxo(input)
```

Comment:
Hey, I'm hoping to be new contributor someday. Just taking a peek at some PRs. Could these original tx assignments be removed since it is being re-assigned?

```cpp
tx = self.create_self_transfer_from_utxo(input)
```

Reply:
Hi reemuru, thanks for taking a look and welcome as new contributor!
Note that the `tx` instance created in this line is used on the right-hand-side of the re-assignment two lines below, i.e. it serves as input for the `sign_tx` method call: `tx = self.miniwallet.sign_tx(tx)`. The best would probably be if `sign_tx` modifies the tx in-place and doesn't return anything, leading to simply `self.miniwallet.sign_tx(tx)` to have cleaner code and reduce confusion. Will check if that is possible.